src/keri/db/dbing.py

```diff
@@ -581,6 +581,37 @@ def getAllItemIter(self, db, key=b'', split=True, sep=b'.'):
                     splits = (bytes(key), val)
                 yield tuple(splits)
 
+    def getAllItemRvsdIter(self, db, key=b'', split=True, sep=b'.'):
```

Comment:
We already have a method that implements something similar to this and is called getIoValsAllPreBackIter.  This name should match.